charts/k8s-monitoring/docs/ScrapeApplicationMetrics.md

```diff
@@ -0,0 +1,205 @@
+
+```river
+  rule {
+    target_label = "team"
+    action = "replace"
+    replacement = "blue"
+  }
+```
+
+### Pod discovery
+
+Similar to service discovery, we use a [`discovery.relabel`](https://grafana.com/docs/agent/latest/flow/reference/components/discovery.relabel/)
+component to select the specific pod or pods that we want to scrape. The [meta labels for pods](https://grafana.com/docs/agent/latest/flow/reference/components/discovery.kubernetes/#pod-role)
+will be slightly different, but the concept is the same.
+
+Here is an example that filters to a specific set of pods that starts with name "analysis", with the label "system.component=image":
+
+```river
+discovery.relabel "image_analysis_pods" {
+  targets = discovery.kubernetes.pods.targets  // Gets all pods
+  rule {  // Keep all pods named "analysis.*"...
+    source_labels = ["__meta_kubernetes_pod_name"]
+    regex = "analysis.*"
+    action = "keep"
+  }
+  rule {  // ... with the label system.component=image
+    source_labels = ["__meta_kubernetes_pod_label_system_component"]
+    regex = "image"
+    action = "keep"
+  }
+}
+```
+
+Note that there is a unique meta label for every Kubernetes label. The labels are prefixed with
+`__meta_kubernetes_pod_label_` and the label name is normalized so all non-alphanumeric characters become underscores (`_`).
+
+## Scraping
+
+Now that we've selected the specific pod or service we want, we can scrape it for metrics. This is done with the
+[`prometheus.scrape`](https://grafana.com/docs/agent/latest/flow/reference/components/prometheus.scrape/) component. At its basic, you only need to declare what things to scrape, and where to send
+the scraped metrics. Here is an example:
+
+```river
+prometheus.scrape "processing_app" {
+  targets = discovery.relabel.image_analysis_pods.targets
+  forward_to = [prometheus.remote_write.grafana_cloud_prometheus.receiver]
+}
+```
+
+This component gives a lot of flexibility to modify how things are scraped, including setting the `job` label, how
+frequently the metrics should be scraped, the path to scrape, and many more. Here is an example with lots of options:
+
+```river
+prometheus.scrape "processing_app" {
+  targets = discovery.relabel.image_analysis_pods.targets
+  job_name = "integrations/processing"
+  scrape_interval = "120s"
+  metrics_path = "/api/v1/metrics"
+  forward_to = [prometheus.remote_write.grafana_cloud_prometheus.receiver]
+}
+```
+
+## Processing
+
+Often, we want to do some post-scrape processing to the metrics that are scraped. Some common reasons are:
+* limiting the amount of metrics being sent up to Prometheus
+* adding labels, changing labels, or dropping labels
+
+Processing is done with the [`prometheus.relabel`](https://grafana.com/docs/agent/latest/flow/reference/components/prometheus.relabel/)
+component. It uses the same type of rules as `discovery.relabel`, but instead of filtering scrape targets, it filters the
+metrics that were scraped.
```

Comment:
```suggestion
component. It uses the same type of rules as `discovery.relabel`, but instead of filtering scrape _targets_, it filters the
_metrics_ that were scraped.
```